lib/resources/orders.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @name Orders
+ * @description This module exposes functions
+ *              related to the `/orders` path.
+ *
+ * @module Orders
```

Comment:
comparado com os demais arquivos do projeto vi que o @module sempre está em minisculo
```suggestion
 * @module orders
```